src/common/utils.ts

```diff
@@ -87,5 +87,5 @@ export const parseDate = (date: string | Date): Date | undefined => {
 };
 
 export const toGQLEnum = (value: Record<string, string>, name: string) => {
-  return `enum ${name} { ${Object.keys(value).join(' ')} }`;
+  return `enum ${name} { ${Object.values(value).join(' ')} }`;
```

Comment:
Make sure it won't break anything 🥵

Reply:
Yeah, actually it should have been like this from beginning.